mods/translucent-windows.wh.cpp

```diff
@@ -542,6 +660,10 @@ void NewWindowShown(HWND hWnd)
     }
     else
         EnableColoredTitlebar(hWnd);
+    
+    if(g_settings.BorderFlag || g_settings.CaptionTextFlag || g_settings.TitlebarFlag)
+        if(WindhawkUtils::SetWindowSubclassFromAnyThread(hWnd, SubclassProc, 0))
+            g_subclassedWindows.insert(hWnd);

```

Comment:
Since you're using a lock:
```suggestion
        {
            std::lock_guard<std::mutex> guard(g_subclassedWindowsMutex);
            g_subclassedWindows.insert(hWnd);
        }
```